platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -174,50 +97,16 @@ struct pktio_entry {
 	} out_queue[PKTIO_MAX_QUEUES];
 };
 
-typedef union {
+union pktio_entry_un {
```

Comment:
Thanks Bill
Here because API declarations in odp_pktio_ops_subsystem.h depends on pktio_entry_t, here I split the typedef as:

/* Forward declaration */
typedef union pktio_entry_un pktio_entry_t;
#include <odp_pktio_ops_subsystem.h>
......
union pktio_entry_un {
......